server/src/config/config.env.js

```diff
@@ -18,4 +18,7 @@ module.exports = {
     username: process.env.JIRA_USERNAME,
     password: process.env.JIRA_PASSWORD,
   },
+  finance: {
+    key: process.env.FINANCE_KEY,
```

Comment:
Please add to .env.sample and include this:

`# Stock ticker to check, e.g. FRA:RKET for Rocket Internet or ETR:DHER for Delivery Hero.`

We should also change this config value to STOCK_TICKER as FINANCE_KEY isn't very specific.